testing/unit/report/report_test.py

```diff
@@ -95,34 +140,57 @@ def report_formatting(self):
                   encoding='utf-8') as f:
                   f.write(formatted_report)
 
-  def add_formatting(self, body):
-    return f'''
+  def add_html_formatting(self, body):
+    """Wrap the raw report inside a complete HTML structure with styles"""
+
+    # Load the css file
+    with open(CSS_PATH, 'r', encoding='UTF-8') as css_file:
+      styles = css_file.read()
+
+    # Define the html template
+    html_template = '''
     <!DOCTYPE html>
     <html lang="en">
-    {TestReport().generate_head()}
+    <head>

```

Comment:
Can you put this header into a html file and load it in that way?

Reply:
Updated to use regex to extract the head from 'test_report_template.html' file